### PR TITLE
refactor: remove unnecessary css

### PR DESCRIPTION
**Pull request by DariusZdroba**
### Description

The CSS was really cluttered and difficult to understand being in the HTML templates in `<style>` tags, I separated them into their own file, now we inject them the same way we did so far with **IaC**, through the TemplateData that is used to dynamically render variables into the HTML.

In addition to that, some minor styling changes were pushed down from **VSCode** in order to remove them from being plugin-specific and having a more centralized styling possibility in **Language Server**.

### Checklist

- [x] Tests added and all succeed
- [x] Linted
- [ ] README.md updated, if user-facing
- [ ] License file updated, if new 3rd-party dependency is introduced

### CODE REVIEW

File: infrastructure/iac/template/styles.css [hunk 1]
```diff
@@ -20,7 +21,7 @@
 }
 
 ::-webkit-scrollbar-thumb:hover {
-  background: var(--scroll-bar-hover-background-color);
+  background: var();
```
Comment (ShawkyZ) [inline infrastructure/iac/template/styles.css:24]: don't we need the var --scroll-bar-hover-background-color here

File: infrastructure/iac/template/styles.css [hunk 2]
```diff
@@ -30,8 +31,9 @@
 }
 
 body {
+  padding: 0;
   background-color: var(--background-color);
-  font-family: var(--default-font);
+  font-family: var(--default-font), sans-serif;
```
Comment (ShawkyZ) [inline infrastructure/iac/template/styles.css:36]: I think this sans-serif is not needed?

File: infrastructure/oss/template/styles.css [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+  color: var(--link-color);
+  cursor: pointer;
+  text-decoration: none;
+}
+
+body {
+  padding: 0;
+  background-color: var(--background-color);
+  color: var(--text-color);
+  font-family: var(--default-font), sans-serif;
```
Comment (ShawkyZ) [inline infrastructure/oss/template/styles.css:41]: same here, shouldn't --default-font be enough?

**Review by ShawkyZ — APPROVED** [review]

File: infrastructure/iac/template/styles.css (hunk 1 above)
Comment (DariusZdroba) [reply infrastructure/iac/template/styles.css:24]: We don't replace it anywhere in IntelliJ or VSCode I'm not sure it's actually used, maybe I'm missing something? Or maybe we use it in VS or Eclipse ?

**Review by Missjones2 — COMMENTED** [review]
Merge

**Pull request merged**